node-js-basics-101/node_modules/ws/lib/Validation.js

```diff
@@ -0,0 +1,17 @@
+/*!
+ * ws: a node.js websocket client
+ * Copyright(c) 2011 Einar Otto Stangvik <einaros@gmail.com>
+ * MIT Licensed
+ */
+
+'use strict';
+
+try {
+  const isValidUTF8 = require('utf-8-validate');
+
+  module.exports = typeof isValidUTF8 === 'object'
+    ? isValidUTF8.Validation.isValidUTF8  // utf-8-validate@<3.0.0
```

Comment:
__[jshint]__
* Misleading line break before '?'; readers may interpret this as an expression boundary.

<!-- Comment by [SideCI](https://sideci.com) -->